Buy/Generated/Storefront/CheckoutAttributesUpdateInput.swift

```diff
@@ -47,25 +47,59 @@ extension Storefront {
 		///     - customAttributes: A list of extra information that is added to the checkout.
 		///     - allowPartialAddresses: Allows setting partial addresses on a Checkout, skipping the full validation of attributes. The required attributes are city, province, and country. Full validation of the addresses is still done at complete time. 
 		///
-		public init(note: String? = nil, customAttributes: [AttributeInput]? = nil, allowPartialAddresses: Bool? = nil) {
+		public static func create(note: Input<String> = .undefined, customAttributes: Input<[AttributeInput]> = .undefined, allowPartialAddresses: Input<Bool> = .undefined) -> CheckoutAttributesUpdateInput {
+			return CheckoutAttributesUpdateInput(note: note, customAttributes: customAttributes, allowPartialAddresses: allowPartialAddresses)
+		}
+
+		private init(note: Input<String> = .undefined, customAttributes: Input<[AttributeInput]> = .undefined, allowPartialAddresses: Input<Bool> = .undefined) {
 			self.note = note
 			self.customAttributes = customAttributes
 			self.allowPartialAddresses = allowPartialAddresses
 		}
 
+		/// Creates the input object.
+		///
+		/// - parameters:
+		///     - note: The text of an optional note that a shop owner can attach to the checkout.
+		///     - customAttributes: A list of extra information that is added to the checkout.
+		///     - allowPartialAddresses: Allows setting partial addresses on a Checkout, skipping the full validation of attributes. The required attributes are city, province, and country. Full validation of the addresses is still done at complete time. 
+		///
+		@available(*, deprecated)
```

Comment:
Shouldn't we provide the reason of deprecation, and mention to use `create` instead?

Reply:
Yeah, you're right. Added.